doc/notebooks/grass_jupyter.ipynb

```diff
@@ -30,46 +26,45 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import os\n",
+    "import subprocess\n",
+    "import sys\n",
+    "\n",
+    "# Add GRASS to path\n",
     "gisbase = subprocess.check_output([\"grass\", \"--config\", \"path\"], text=True).strip()\n",
     "os.environ[\"GISBASE\"] = gisbase\n",
-    "sys.path.append(os.path.join(gisbase, \"etc\", \"python\"))"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
+    "sys.path.append(os.path.join(gisbase, \"etc\", \"python\"))\n",
+    "\n",
+    "# Import GRASS packages\n",
     "import grass.script as gs\n",
-    "import grass.jupyter as gj"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
+    "import grass.jupyter as gj\n",
+    "\n",
     "# Start GRASS Session\n",
-    "gj.init(\"../../data/grassdata\", \"nc_basic_spm_grass7\", \"user1\")"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# Set computational region to the study area.\n",
+    "gj.init(\"../../../grassdata\", \"nc_basic_spm_grass7\", \"user1\")\n",
```

Comment:
It does not work here with this path. Why not the one that was before? If I switch back to that one, it runs fine. Otherwise, errors because it does not find the data

Reply:
Yes, the path should be `../../data/grassdata` - I changed it to test on my local machine before pushing and forgot to change it back. Ooops! It'll be fixed on the next commit.

Reply:
I think I marked the conversation as resolved too soon. The path still needs fixing.

Related to that, I guess another future todo is to fail in gj.init when the mapset does not exist...which brings in another issue and that is a good way of creating a location.

Reply:
Noted. Yes, I fixed it in one commit then changed it back in the next one to test it on my computer and then forgot to change it back again... Oops.